geometry_msgs/Pose pose
int32 id
bool is_big
bool detect_success